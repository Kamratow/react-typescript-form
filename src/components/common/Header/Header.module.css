.Header {
    font-size: 40px;
    margin-left: 30px;
}
