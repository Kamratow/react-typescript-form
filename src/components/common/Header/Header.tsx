import React from "react";

import styles from "./Header.module.css";

export interface IHeaderProps {
    headerContent: string;
}

const Header: React.FunctionComponent<IHeaderProps> = ({ headerContent }) => {
    return <h1 className={styles.Header}>{headerContent}</h1>;
};

export default Header;
