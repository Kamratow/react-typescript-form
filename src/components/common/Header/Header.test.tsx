import React from "react";
import { shallow } from "enzyme";

import Header, { IHeaderProps } from "./Header";

const defaultProps: IHeaderProps = {
    headerContent: "test header content",
};

describe("Header", () => {
    it("should render properly with default props", () => {
        const wrapper = shallow(<Header {...defaultProps} />);

        expect(wrapper).toMatchSnapshot();
    });
});
