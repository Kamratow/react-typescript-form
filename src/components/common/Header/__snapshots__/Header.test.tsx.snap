// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly with default props 1`] = `
<h1
  className="Header"
>
  test header content
</h1>
`;
